crates/sui-config/src/node.rs

```diff
@@ -101,6 +101,34 @@ pub struct NodeConfig {
     pub supported_protocol_versions: Option<SupportedProtocolVersions>,
 }
 
+impl NodeConfig {
+    /// Zero the ip address to ensure we bind to `0.0.0.0`.
+    pub fn zero_ip_address(&mut self) {
```

Comment:
I would try to give a bit more specific name to this method, ex `zero_network_adresses`